packages/cli/internal/pkg/cli/config/config_client.go

```diff
@@ -135,3 +135,20 @@ func (c Client) GetUserId() (string, error) {
 	userId := userIdFromEmailAddress(userEmailAddress)
 	return userId, nil
 }
+
+func (c Client) SetFormat(format string) error {
+	configData, err := c.loadFromFile()
+	if err != nil {
+		return err
+	}
+	configData.Format.Format = format
+	return c.storeToFile(configData)
+}
+
+func (c Client) GetFormat() (string, error) {
+	configData, err := c.loadFromFile()
+	if err != nil {
+		return "", err
+	}
+	return configData.Format.Format, nil
```

Comment:
To my previous point. Don't you think this `...Format.Format...` repetition looks and feels weird? It would look better if we have something like `...Format.Name...`, but still not convinced that just simple `configData.Format` will not suffice.

Reply:
Agreed. I will update that to something more readable